src/addons/dragAndDrop/EventWrapper.js

```diff
@@ -32,28 +23,28 @@ class EventWrapper extends React.Component {
 
   handleResizeUp = e => {
     if (e.button !== 0) return
-    e.stopPropagation()
     this.context.draggable.onBeginAction(this.props.event, 'resize', 'UP')
   }
   handleResizeDown = e => {
     if (e.button !== 0) return
-    e.stopPropagation()
     this.context.draggable.onBeginAction(this.props.event, 'resize', 'DOWN')
   }
   handleResizeLeft = e => {
     if (e.button !== 0) return
-    e.stopPropagation()
```

Comment:
It seems like removing these would create bugs, not fix them? Can you walk me through why this is necessary?

Reply:
Ok, this one is tricky. the Dnd addon uses rbc's selectable() mechanism to monitor mouse events before/during drags. selectable needs to see those events in order to work. The event wrapper stopped the propagation and so the selection logic was basically broken.

So first, I removed the stop propagation - that helped a lot.

But because the resize anchors are on top of the drag (move) anchor, clicking on a resize would also start a drag/move, which would abort the resize, etc.

So I added a check (see :42) to ignore this case:

```
    // hack: because of the way the anchors are arranged in the DOM, resize
    // anchor events will bubble up to the move anchor listener. Don't start
    // move operations when we're on a resize anchor.
    const isResizeHandle = e.target.className.includes('rbc-addons-dnd-resize')
    if (!isResizeHandle)
      this.context.draggable.onBeginAction(this.props.event, 'move')
```

Now the million dollar question is _why did this ever work?_ My only theory is that react 17 changed where the event listeners were placed and _maybe_ that subtle changed the even propagation?

I would love to back all of this out, but it seemed to be required both in reading the code and trying it.

Reply:
sounds like you got it all handled 👍